packages/components/src/components/input/input.ts

```diff
@@ -790,6 +790,11 @@ export default class SdInput extends SolidElement implements SolidFormControl {
         caret-color: transparent;
       }
 
+      :host([visually-disabled]) ::placeholder,
+      :host([disabled]) ::placeholder {
+        @apply text-neutral-500;
+      }
+
```

Comment:
Should be possible inline, right? https://tailwindcss.com/docs/hover-focus-and-other-states#placeholder

Reply:
Updated